toolchain/sem_ir/name_scope.h

```diff
@@ -51,6 +52,7 @@ class NameScope : public Printable<NameScope> {
 
   // Searches for the given name and returns an EntryId if found or nullopt if
   // not.
+  //
   // If the name is found, it might be a poisoned name.
```

Comment:
```suggestion
  // Searches for the given name and returns an EntryId if found or nullopt if
  // not. The returned entry may be poisoned.
```

I'm going to suggest using fewer paragraphs. The context and briefer read can often be helpful, and paragraphs are typically a few sentences anyways.

Reply:
Done.